plugin/forward/setup.go

```diff
@@ -144,7 +144,8 @@ func parseStanza(c *caddy.Controller) (*Forward, error) {
 		}
 		f.proxies[i].SetExpire(f.expire)
 		f.proxies[i].health.SetRecursionDesired(f.opts.hcRecursionDesired)
-		if f.opts.forceTCP {
+		// when TLS is used, checks are set to tcp-tls
+		if f.opts.forceTCP && transports[i] != transport.TLS {
```

Comment:
Good catch

Reply:
As you mentioned TLS, I better check that.